Python/initconfig.c

```diff
@@ -173,8 +173,8 @@ Options (and corresponding environment variables):\n\
          also PYTHONWARNINGS=arg\n\
 -x     : skip first line of source, allowing use of non-Unix forms of #!cmd\n\
 -X opt : set implementation-specific option\n\
---check-hash-based-pycs always|default|never:\n\
-         control how Python invalidates hash-based .pyc files\n\
+--check-hash-based-pycs arg: control how Python invalidates hash-based .pyc\n\
+         files; arg is always|default|never\n\
```

Comment:
"argparse" style is more compact:
```suggestion
--check-hash-based-pycs {always|default|never}:\n
         control how Python invalidates hash-based .pyc files\n\
```